src/media/css/rtl/footer.styl

```diff
@@ -0,0 +1,10 @@
+@import '../lib';
+
+@media $base-tablet {
+    +rtl() {
+        .footer p {
+            padding: 0 1em 0 0;
+            margin: 0 0 0 100px;
```

Comment:
Yeah, the spacing doesn't seem like it needs to stay the way it is. It changes a bit but it still looks good.